### DISCUSSION TITLE
[Announcement] HW3 and HW4 graded

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#350] [Announcements]
Hi all,
HW 3 and 4 is graded.

You can check your grade in KLMS, Week 12. We made a dummy assignment "HW3" and "HW4", so you can check out your personal score there. Note that this is the score for which the late submission penalty is applied.

We also pushed the additional test cases used for the grading in your personal homework repositories.
You can run it by yourself and find out what test you have failed (if you failed any).

For HW3, you can check the details of your score in Gradescope. You will be able to find the detailed criteria for your score. Note that the scores in Gradescope are the score with no late submission penalty. You must check the KLMS to find out your finalized score. If you have turned in your assignment on time, checking Gradescope will be sufficient.

Feel free to contact us if you have any questions. ([cs524.ta@prosys.kr](mailto:cs524.ta@prosys.kr))

Regards,
Tae Eun.

**Comment by goodtaeeun** [comment]
There has been a mistake in giving the late penalty.
As a result, some of your scores will be increased.

We will update your scores in KLMS as soon as we can (currently not able to access it...)
Sorry for the inconvenience.

Regards,
Tae Eun

**Comment by KAIST-JongchanPark** [comment]
Now, Scores in KLMS are updated.
